src/store/substrate/event-types.json

```diff
@@ -5,15 +5,15 @@
         "OrderPaid": {
           "message": "",
           "value": "id",
-          "value_message": "You've successfully submitted your requested test ",
+          "value_message": "Congrats! You successfully submitted your requested test for ",
           "route": "customer-payment-details",
           "params": "id",
           "identity": "customerId"
         },
         "OrderFulfilled": {
           "message": "",
           "value": "id",
-          "value_message": "Congrats! Your test results ",
+          "value_message": "Congrats! Your test results for ",
```

Comment:
check notification wording on validation sheet in column #Revision2 or the very right of the column